file-formats/ront/type/zif_aff_ront_v1.intf.abap

```diff
@@ -0,0 +1,60 @@
+INTERFACE zif_aff_ront_v1
+  PUBLIC.
+
+  CONSTANTS:
+    BEGIN OF co_category,
+      "! <p class="shorttext">Business Object</p>
+      "! Business object
+      business_object      TYPE c LENGTH 2 VALUE `BO`,
+      "! <p class="shorttext">Technical Object</p>
+      "! Technical object
+      technical_object     TYPE c LENGTH 2 VALUE `TO`,
+      "! <p class="shorttext">Analytical Object</p>
+      "! Analytical object
+      analytical_object    TYPE c LENGTH 2 VALUE `AO`,
+      "! <p class="shorttext">Configuration Object</p>
+      "! Configuration object
+      configuration_object TYPE c LENGTH 2 VALUE `CO`,
+    END OF co_category.
+
+  TYPES:
+    "! <p class="shorttext">Node</p>
+    BEGIN OF ty_rontnode,
+      "! <p class="shorttext">Name</p>
+      "! Raw (camel case) node name
+      "! $required
+      node_name_raw TYPE c LENGTH 40,
```

Comment:
I like that you reduced it to just one node name. Thanks. 

If this is the external representation, would you also like to keep the `_raw` in the end or does it make sense to call it just `node_name`?

Reply:
Yes, that makes sense to me. I would suggest to use `name` instead of `node_name`, however, as this avoids potential confusion between external and internal node names in the persistence coding for object type RONT.